### ISSUE TITLE
Spec for execute_result doesn't mention transient field

### ISSUE THREAD

**Issue by takluyver** [jupyter/jupyter_client#305]
The [spec](http://jupyter-client.readthedocs.io/en/latest/messaging.html#id6) for `execute_result` messages says:

> These are identical to display_data messages, with the addition of an execution_count key.

However, `display_data` has an optional `transient` field, which is not mentioned in the description of `execute_result`. Is it allowed there? Or is that field only for display_data and update_display_data?

**Comment by rgbkrk** [comment]
execute_result should be able to have the transient field, though I don’t know of a direct way to use it in ipython (especially for setting a display id)

**Comment by minrk** [comment]
yeah, the "identical to display_data" statement should take precedence, since it is the *definition* as opposed to the description. I just didn't copy the transient part when updating everything else.

**Comment by takluyver** [comment]
Thanks. I'm just updating jupyter_kernel_test to validate messages more strictly.
